share/picongpu/examples/FoilLCT/include/picongpu/param/particle.param

```diff
@@ -62,7 +62,7 @@ namespace manipulators
         )
         {
             constexpr float_X protonNumber = GetAtomicNumbers< T_Particle >::type::numberOfProtons;
-            particle[ boundElectrons_ ] = protonNumber - 1;
+            particle[ boundElectrons_ ] = protonNumber - 1._X;
```

Comment:
the idea is, that you can write `1_X` even ;-)
```suggestion
            particle[ boundElectrons_ ] = protonNumber - 1_X;
```

Reply:
uh, turns out I did not overload that for integers yet xD
So you will have to write `1._X` indeed or maybe more readable, `1.0_X` :)